element_array_ephys/probe.py

```diff
@@ -78,15 +96,16 @@ def create_neuropixels_probe(probe_type='neuropixels 1.0 - 3A'):
         def build_electrodes(site_count, col_spacing, row_spacing,
                              white_spacing, col_count,
                              shank_count, shank_spacing):
-            """
-            :param site_count: site count per shank
-            :param col_spacing: (um) horrizontal spacing between sites
-            :param row_spacing: (um) vertical spacing between columns
-            :param white_spacing: (um) offset spacing
-            :param col_count: number of column per shank
-            :param shank_count: number of shank
-            :param shank_spacing: spacing between shanks
-            :return:
+            """Builds electrode layouts.
+
+            Args:
```

Comment:
Please add typing suggestions to functions def. e.g., def func(arg1: int, arg2: str = "DefaultStr")

If you take this step first in the future, you can get templated args with [autoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) e.g., `arg2 (str): _description_. Defaults to 'DefaultStr'` autopopulates

Reply:
Done. And thank you, I will use that in the future